pages/releases/4.3.html

```diff
@@ -0,0 +1,2158 @@
+				<h3>
+					<a href="#special-thanks">Special thanks</a>
+					<div class="section-robot godot-icon-godot" aria-disabled="true"></div>
+				</h3>
+			</div>
+		</div>
+		<div class="container">
+			<div class="release-cards">
+				<div id="special-thanks-release-authors" class="release-card span-3 media-bottom">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#special-thanks-release-authors">
+										Release authors
+									</a>
+								</h4>
+								<div class="c-blockquote">
+									<span class="highlight">Team work</span> makes the dream work
+								</div>
+								<div class="c-content">
+									<p>
+										Here are all the contributors that saw their PRs merged, ordered alphabetically,
+										and sized accordingly to their number of merged PRs.
+									</p>
+									<p>
+										Also special thanks to our teams (communications, documentation, godot-cpp,
+										and translations to name a few) and to everyone that contributed in any form
+										to this release.
+									</p>
+								</div>
+							</div>
+						</div>
+						<div class="release-card-media">
+							<div class="release-card-authors">
+								{% assign authors = page.authors_list | split: "|" %}
+								{% for author in authors %}
+								{% assign author_data = author | split: "," %}
+								{% assign author_handle = author_data[0] %}
+								{% assign author_commits = author_data[1] %}
+								{% assign author_prs = author_data[2] %}
+								<div class="release-card-author" data-handle="{{author_handle}}"
+									data-commits="{{author_commits}}" data-prs="{{author_prs}}">
+									<a href="https://github.com/godotengine/godot/pulls?q=author%3A{{author_handle}}+is%3Apr+milestone%3A4.3+is%3Aclosed+is%3Amerged+"
+										target="_blank">{{author_handle}}</a>
+								</div>
+								{% endfor %}
+							</div>
+						</div>
+					</div>
+				</div> <!-- #special-thanks-release-authors -->
+
+				<div id="special-thanks-credits" class="release-card span-3 media-bottom">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#special-thanks-credits">
+										Release page credits
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										Design by <a href="https://luiscarli.com/">Luis Carli</a>.
+									</p>
+									<p>
+										Integration, demo creation, footage capture, and video editing by <a href="https://adamscott.studio">Adam Scott</a>. Additional help by <a href="https://coppolaemilio.com/">Emilio Coppola</a>.
+									</p>
+									<p>
+										Shout-out to <a href="https://gdquest.com/">GDQuest</a> for their <a href="#outstanding-new-tilemaplayer-node"><code>TileSetLayer</code> footage</a>.
+									</p>
```

Comment:
```suggestion
								<div class="c-content">
								        <p>
										Coded by <a href="https://adamscott.studio">Adam Scott</a> & <a href="https://coppolaemilio.com/">Emilio Coppola</a>
									</p>
									<p>
										Design template by <a href="https://luiscarli.com/">Luis Carli</a>
									</p>
									<p>
										Visual content by <a href="https://adamscott.studio">Adam Scott</a> with special thanks to <a href="https://gdquest.com/">GDQuest</a>
									</p>
									<p>
										Texts by <a href="https://bio.link/murderveggie">Nathalie Galla</a> & <a href="https://adamscott.studio">Adam Scott</a> 
									</p>
```

Reply:
I added [Kenney](https://kenney.nl/) in the special thanks, as many assets comes from him.